Orbitersdk/samples/ProjectApollo/src_sys/nasspdefs.h

```diff
@@ -211,7 +211,7 @@
 #define stricmp _stricmp
 #endif 
 
-#define NASSP_VERSION 80000		///< Current NASSP version.
+#define NASSP_VERSION 80001		///< The version of the current scenario file. The first two digits are reserved to indicate the current NASSP version. The remaining 3 digit number is incremented each time a breaking change is made to a scenario parameter
```

Comment:
You also need to update NASSPVER in the scenarios. Otherwise all scenarios will trigger the warning

Reply:
That should be an easy fix. I can do this tonight. Thank you.

Reply:
Also you might want to link to https://nassp.space/index.php/Scenario_File_Options instead of pasting the description from it.

Reply:
This should be complete now. I elected not to update the 70000s in the Apollo 7 Scenarios, since there _are_ a bunch of 7.0 vs 8.0 semi-breaking changes that should be fixed with new Apollo 7 Scenarios.